packages/big-design/src/components/AccordionGroup/AccordionPanel/styled.tsx

```diff
@@ -0,0 +1,60 @@
+import { theme as defaultTheme, remCalc } from '@bigcommerce/big-design-theme';
+import styled, { css } from 'styled-components';
+
+import { withTransition } from '../../../mixins/transitions';
+import { Box } from '../../Box';
+import { StyleableButton } from '../../Button/private';
+
+import { AccordionPanelProps } from './index';
+
+export const StyledAccordionButton = styled(StyleableButton)<AccordionPanelProps>`
```

Comment:
Should we create an interface for the `StyledAccordionButton` to define the props that is gonna use instead of passing the `<AccordionPanelProps>` ? 

Something like this:

```tsx
interface StyledAccordionButtonProps {
  isExpanded: boolean;
}

StyledAccordionButton = styled(StyleableButton)<StyledAccordionButtonProps>`
```